packages/windows/src/RPA/Windows/utils.py

```diff
@@ -1,27 +1,5 @@
-from typing import Dict
 import platform
-
-IS_WINDOWS = platform.system() == "Windows"
-
-if IS_WINDOWS:
-    import psutil
-
-
-def get_process_list() -> Dict:
-    """Get process list.
-
-    Returns dictionary mapping process id to process name
-    """
-    return {proc.pid: proc.name() for proc in psutil.process_iter()}
-
-
-def is_numeric(value):
-    try:
-        float(value)
-    except ValueError:
-        return False
-    else:
-        return float(value).is_integer()
+from RPA.core.windows.helpers import IS_WINDOWS, get_process_list, is_numeric
```

Comment:
Importing these here just to preserve the old behaviour of `Windows.utils`, place where we had these before moving them into **core**. (other modules are depending on these from here)